target/linux/ramips/dts/mt7628an_wiznet_wizfi630s.dts

```diff
@@ -172,13 +154,10 @@
 
 &esw {
 	mediatek,portmap = <0x3e>;
+	mediatek,portdisable = <0x26>;
```

Comment:
That's not a pin change.

Reply:
WizFi630S is using only 3 phy ports - so the unneded 2 are disabled to save power. I'll put it to a separate commit.

Reply:
Yes, but IIRC the driver only takes three hard-coded values, 0x3e, 0x2f and a third one I don't remember.

Reply:
https://github.com/openwrt/openwrt/pull/2667

Reply:
> Yes, but IIRC the driver only takes three hard-coded values, 0x3e, 0x2f and a third one I don't remember.

https://github.com/openwrt/openwrt/pull/2667 is basically telling that mt7530 is accepting only three values. 

From what i know it's a binary map. `root@OpenWrt:/# swconfig dev switch0  show | grep Port -A 1
Port 0:
        disable: 0
--
Port 1:
        disable: 1
--
Port 2:
        disable: 1
--
Port 3:
        disable: 0
--
Port 4:
        disable: 0
--
Port 5:
        disable: 1
--
Port 6:
        disable: 0
root@OpenWrt:/#
` At runtime the portmap resolves to the expected values in the switchconfig. The module is using Port 0, 3, 4 and 6 of course.

Reply:
The hard-coded one is the portmap for mt7530. The portdisable property is a binary map, right.

Reply:
Sorry, I mixed up portmap and portdisable here, didn't look properly. Thanks for verifying on device nevertheless!